hack/make/ubuntu

```diff
@@ -91,6 +95,10 @@ if [ "$1" = 'configure' ] && [ -z "$2" ]; then
 	fi
 fi
 
+if [ -x /sbin/apparmor_parser ]; then
+	/sbin/apparmor_parser -r -W -T /etc/apparmor.d/ docker
```

Comment:
The space is intentional.

The 'aa-status --enabled' is entirely reasonable. Libcontainer was using the check for the binary, which is a terrible way to check, but I was keeping for a minimal change.